tests/e2e-pw/specs/merchant/merchant-admin-account-balance.spec.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { useMerchant } from '../../utils/helpers';
+
+test.describe(
+	'Merchant account balance overview for single deposit currency accounts',
+	() => {
+		// Use the merchant user for this test suite.
+		useMerchant();
+
+		test(
+			'View the total and available account balance for a single deposit currency (USD)',
+			{
+				tag: '@critical',
+			},
```

Comment:
@alopezari it would be great to get your perspective on the use of tags (`@critical` and `@todo`) and `test.step` in this PR – to see if it aligns with your RFC: _Simplifying the measurement and tracking of E2E critical flows coverage_ (paJDYF-dVT-p2).